java/openjdk/Portfile

```diff
@@ -268,6 +321,8 @@ subport openjdk11-openj9-large-heap {
 
 subport openjdk11-sap {
     # https://github.com/SAP/SapMachine/releases/
```

Comment:
Change `# https://github.com/SAP/SapMachine/releases/` to `# https://sap.github.io/SapMachine/latest/11`